plan/stats.go

```diff
@@ -49,6 +49,7 @@ func (s *statsInfo) scale(factor float64) *statsInfo {
 }
 
 // We try to scale statsInfo to an expectCnt which must be smaller than the derived cnt.
+// TODO: try to use a better way to do this.
```

Comment:
need to add a issue to this TODO and describe it in detail.

Reply:
This should be in proposal of cost model(or another doc that introduce the `statsInfo`). I think don't need a another issue.